cogs/cipher.py

```diff
@@ -0,0 +1,129 @@
+import base64
+import hashlib
+import re
+import string
+
+from discord.ext import commands
+
+import cogs.utils.parser as parser
```

Comment:
```suggestion
import string

from discord.ext import commands

```
I deleted the part where those two were useful